examples/text_classification_with_scriptable_tokenzier/handler.py

```diff
@@ -0,0 +1,107 @@
+"""
+Module for text classification with scriptable tokenizer
+DOES NOT SUPPORT BATCH!
+"""
+from abc import ABC
+import logging
+
+import torch
+import torch.nn.functional as F
+# Necessary to successfully load the model (see https://github.com/pytorch/text/issues/1793)
+import torchtext  # pylint: disable=unused-import
+
+from ts.torch_handler.base_handler import BaseHandler
+from ts.utils.util import map_class_to_label, CLEANUP_REGEX
+
+
+logger = logging.getLogger(__name__)
+
+def remove_html_tags(text):
+    """
+    Removes html tags
+    """
+    clean_text = CLEANUP_REGEX.sub("", text)
+    return clean_text
+
+class TextClassifier(BaseHandler, ABC):
```

Comment:
Was the `text_handler` not the best option for you? Could you elaborate a bit more on why?

Reply:
The `text_handler` forces me to load a [source_vocab](https://github.com/pytorch/serve/blob/a4d5090e114cdbeddf5077a817a8cd02d129159e/ts/torch_handler/text_handler.py#L53) and [loads a tokenizer](https://github.com/pytorch/serve/blob/a4d5090e114cdbeddf5077a817a8cd02d129159e/ts/torch_handler/text_handler.py#L37) which is not necessary in my case.